docs/archived/locked-inc-ftp/figures/firmware/source/tasks/led_task.c

```
@@ -64,25 +63,26 @@ static esp_err_t internal_update_led_from_sensor_data(led_task_context_t* const
-  shared_context_t* shared_ctx = shared_data_get_context();
+static void internal_process_sensor_data_queue(led_task_context_t *const ctx) {
+  shared_context_t *shared_ctx = shared_data_get_context();
   if (shared_ctx == NULL || shared_ctx->sensor_data_queue == NULL) {
     return;
   }
 
-  sensor_data_t  sensor_data;
-  uint32_t       processed     = 0;
+  sensor_data_t sensor_data;
+  uint32_t processed = 0;
   const uint32_t max_per_cycle = 8; /* Limit processing to prevent starvation */
```

Clang (14.0.6)

[warning] variable 'max_per_cycle' is not initialized

(cppcoreguidelines-init-variables)